dokka-subprojects/plugin-base/src/test/kotlin/markdown/LinkTest.kt

```diff
@@ -121,6 +123,105 @@ class LinkTest : BaseAbstractTest() {
         }
     }
 
+    @Test
+    @OnlySymbols("#3207")
```

Comment:
Will these tests not pass with K1? Is the behaviour still different in K1 and K2? I thought we were going to make it work the same way as in K1, so that the common tests pass

Reply:
#3207: In Dokka K1 `[this]` has an incorrect link that leads to a page of containing package.  For example, `fun pkcg.SomeClass.f()` a `[this]` link references to the `pkcg` page.